### ISSUE TITLE
bbox in demo

### ISSUE THREAD

**Issue by linuxianLoord** [Scalsol/mega.pytorch#56]
Hi,

I have an issue in running the demo, there's no error and the video is outputed in vizualisations folder but without any bbox.
Could you give me some cue?

Thanks

**Comment by fengyue20** [comment]
I have this issue too. Did you change the {CONFIG_FILE} to 'configs/MEGA/vid_R_101_C4_MEGA_1x.yaml'

**Comment by khadijakhaldi** [comment]
please did you solve this issue ?

**Comment by adityamd** [comment]
Is the issue solved?

**Comment by returnlsz** [comment]
yeah,maybe I have found what's wrong with that,in your script demo.py,you can set the score threshold which is default 0.7(since your model's predcition is low confidence then will  be discard so you won't see any box in results)
